pkg/agent/openflow/multicast_test.go

```diff
@@ -80,3 +86,61 @@ func Test_featureMulticast_initFlows(t *testing.T) {
 		})
 	}
 }
+
+// If any test case fails, please consider setting binding.MaxBucketsPerMessage to a smaller value.
+func TestMulticastReceiversGroupMaxReceiversPerOpenflowMessage(t *testing.T) {
```

Comment:
nit: I would just call it `TestMulticastReceiversGroupMaxBuckets`

Reply:
Much better!